uint32 TEST = 0
uint32 LOWER = 1
uint32 MOVE = 2
uint32 LAX_MOVE = 3

# Define the goal
uint32 command
string point_of_interest
string center_point
---
# Define the result
---
# Define a feedback message
